app/src/features/reports/TellMeAboutReport.tsx

```diff
@@ -0,0 +1,151 @@
+import { MadLib, PhraseSelections } from "../../utils/MadLibs";
+import Select from "@material-ui/core/Select";
+import FormControl from "@material-ui/core/FormControl";
+import MenuItem from "@material-ui/core/MenuItem";
+
+/*
+Corresponds to:
+Tell me about {0:"copd", 1:"diabetes"} in USA ?
+*/
+
+interface County {
+  id: string;
+  name: string;
+  value: number;
+}
+
+function CountyLevelTable(countyList: County[]) {
+  return (
+    <TableContainer component={Paper}>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell>Name</TableCell>
+            <TableCell>Cases</TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {countyList.map((county: County) => (
+            <TableRow key={county.name}>
+              <TableCell>{county.name}</TableCell>
+              <TableCell>{county.value}</TableCell>
+            </TableRow>
+          ))}
+        </TableBody>
+      </Table>
+    </TableContainer>
+  );
+}
+
+function TellMeAboutReport(props: {
+  madlib: MadLib;
+  phraseSelections: PhraseSelections;
+}) {
+  const [countyList, setCountyList] = useState<County[]>([]);
+  const [race, setRace] = useState<string>("All");
+
+  useEffect(() => {
+    setCountyList([]);
+    setRace("All");
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [props.phraseSelections[1]]);
+
+  const signalListeners: any = {
+    click: (...args: any) => {
+      const clickedData = args[1];
+      let countyIds = countyList.map((datum: County) => datum.id);
+      if (!countyIds.includes(args[1].id)) {
+        console.log(clickedData);
+        let newCountyDatum = {
+          id: clickedData.id,
+          name: clickedData.properties.name,
+          value: clickedData[FIELDS[props.phraseSelections[1]].field],
+        };
+        setCountyList([...countyList, newCountyDatum]);
+      }
+    },
+    shiftClick: () => {
+      setCountyList([]);
+    },
+  };
+
+  const FIELDS: Record<number, any> = {
+    0: { field: "COPD_YES", legend: "# COPD cases" },
+    1: { field: "DIABETES_YES_YESPREGNANT", legend: "# Diabetes cases" },
+  };
+
+  const RACES = [
+    "All",
+    "American Indian/Alaskan Native, Non-Hispanic",
+    "Asian, Non-Hispanic",
+    "Black, Non-Hispanic",
+    "Hispanic",
+    "Other race, Non-Hispanic",
+    "White, Non-Hispanic",
+  ];
+
+  return (
+    <Grid container spacing={1} alignItems="flex-start">
+      <Grid item xs={12}>
+        <h2>
+          {props.madlib.phrase.map((phraseSegment, index) => (
+            <React.Fragment key={index}>
+              {phraseSegment.constructor === Object ? (
+                <span> {phraseSegment[props.phraseSelections[index]]} </span>
+              ) : (
+                <span>{phraseSegment}</span>
+              )}
+            </React.Fragment>
+          ))}
+        </h2>
+      </Grid>
+      <Grid item xs={12} sm={12} md={6}>
+        Filter results by race:
+        <FormControl>
+          <Select
```

Comment:
We'll probably want to generalize the different filter UI components at some point

Reply:
Probably. I'll wait till we get more guidance on what exactly product wants...

Reply:
Sounds good haha